tests/test_datetime/test_datetime_subtraction.py

```diff
@@ -0,0 +1,402 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from sklearn.pipeline import Pipeline
+
+from feature_engine.creation import RelativeFeatures
```

Comment:
this file has not been updated yet, right?

Reply:
correct, I copied and pasted this from RelativeFeatures